Doc/library/typing.rst

```diff
@@ -1368,47 +1368,38 @@ These can be used as types in annotations. They all support subscription using
       T1 = Annotated[int, ValueRange(-10, 5)]
       T2 = Annotated[T1, ValueRange(-20, 3)]
 
-   Details of the syntax:
+   The first argument to ``Annotated`` must be a valid type. Multiple metadata
+   elements can be supplied as ``Annotated`` supports variadic arguments. The
+   order of the metadata elements is preserved and matters for equality checks::
 
-   * The first argument to ``Annotated`` must be a valid type
-
-   * Multiple metadata elements can be supplied (``Annotated`` supports variadic
-     arguments)::
-
-        @dataclass
-        class ctype:
-            kind: str
-
-        Annotated[int, ValueRange(3, 10), ctype("char")]
-
-     It is up to the tool consuming the annotations to decide whether the
-     client is allowed to add multiple metadata elements to one annotation and how to
-     merge those annotations.
+      @dataclass
+      class ctype:
+           kind: str
 
-   * ``Annotated`` must be subscripted with at least two arguments (
-     ``Annotated[int]`` is not valid)
+      Annotated[int, ValueRange(3, 10), ctype("char")]
```

Comment:
Maybe "ch: Annotated[...]" instead of just creating the Annotated object without even using it?